### PR TITLE
v4 - Fix container option example code

### PR DESCRIPTION
**Pull request by zacechola**
Minor edit to add JS object.

### CODE REVIEW

File: docs/components/popovers.md [hunk 1]
```diff
@@ -43,9 +43,9 @@ When you have some styles on a parent element that interfere with a popover, you
 
 {% highlight js %}
 $(function () {
-  $('.example-popover').popover(
+  $('.example-popover').popover({
     container: 'body'
```
Comment (kkirsche) [inline docs/components/popovers.md]: In an object, should 'container' be in double quotes per JSON specs?

Comment (zacechola) [inline docs/components/popovers.md]: None of the other examples use it. It's just a plain object.

Comment (kkirsche) [inline docs/components/popovers.md]: Ah, ok. That's an oversight on my part. Wasn't fully aware that there was a difference between "JSON" and "Javascript Object Literal". Sorry about that :+1:

**Comment by hnrch02** [comment]
Thanks

**Pull request merged**